{{cookiecutter.__src_folder_name}}/.github/workflows/azure-dev.yml

```diff
@@ -67,3 +69,39 @@ jobs:
             AZURE_ENV_NAME: ${{ vars.AZURE_ENV_NAME }}
             AZURE_LOCATION: ${{ vars.AZURE_LOCATION }}
             AZURE_SUBSCRIPTION_ID: ${{ vars.AZURE_SUBSCRIPTION_ID }}
+
+        - name: Output Deployment URI
+          id: output
+          run: |
+            azd env get-values > .env
+            source .env
+            echo "uri=$BACKEND_URI" >> "$GITHUB_OUTPUT"
+
+    smoketests:
+      runs-on: ubuntu-latest
+      needs: build
+      steps:
+
+        - name: Basic smoke test (curl)
+          env:
+            URI: ${{needs.build.outputs.uri}}
+          run: |
+            echo "Sleeping 1 minute due to https://github.com/Azure/azure-dev/issues/2669"
```

Comment:
I do wonder if we want to do this a fancier way. We could even write a Python script using tenacity to check for 200 status with backoff and 10 attempts, that sort of thing. Or we could do that in bash (barf).

Reply:
I think we can make it cooler later... this works for now (unless it doesn't)

Reply:
It does seem to work so far! We'll see what happens when we cruft update on all 17 repos.